src/Kiota.Builder/Writers/Python/CodeMethodWriter.cs

```diff
@@ -38,7 +38,8 @@ public override void WriteCodeElement(CodeMethod codeElement, LanguageWriter wri
         var requestBodyParam = codeElement.Parameters.OfKind(CodeParameterKind.RequestBody);
         var requestConfigParam = codeElement.Parameters.OfKind(CodeParameterKind.RequestConfiguration);
         var requestParams = new RequestParams(requestBodyParam, requestConfigParam);
-        if (!codeElement.IsOfKind(CodeMethodKind.Setter))
+        if (!codeElement.IsOfKind(CodeMethodKind.Setter) &&
+        !(codeElement.IsOfKind(CodeMethodKind.Constructor) && parentClass.IsOfKind(CodeClassKind.RequestBuilder)))
```

Comment:
```suggestion
        if (!codeElement.IsOfKind(CodeMethodKind.Setter, CodeMethodKind.Constructor) &&
        parentClass.IsOfKind(CodeClassKind.RequestBuilder)))
```

Reply:
I don't think the changes achieve what was intended. 
`!(codeElement.IsOfKind(CodeMethodKind.Constructor) && parentClass.IsOfKind(CodeClassKind.RequestBuilder))` is meant to be one condition that evaluates to true or false.